cep-10.md

```diff
@@ -0,0 +1,90 @@
+<table>
+<tr><td> Title </td><td> Conda Version Support </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>
+<tr><td> Created </td><td> May 19, 2022</td></tr>
+<tr><td> Updated </td><td> March 23, 2023</td></tr>
+<tr><td> Discussion </td><td>  NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CEP introduces an official version support policy to promote transparency and certainty 
+about which versions of conda are supported. The policy itself states that only the latest
+released version of conda will be supported. Instead of patching previous releases, maintainers will only focus their efforts on new and current releases. The implications of
```

Comment:
```suggestion
released version of conda will be supported. Instead of patching previous releases, maintainers
will only focus their efforts on new and current releases. The implications of
```